gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -189,6 +189,14 @@ internal fun AvatarPicker(uiState: AvatarPickerUiState, onEvent: (AvatarPickerEv
                             }
                         },
                         onAltTextSelected = { /* Future implementation */ },
+                        onDeleteSelected = { avatarUi ->
+                            when (avatarUi) {
+                                is AvatarUi.Uploaded -> onEvent(AvatarPickerEvent.AvatarDeleteSelected(avatarUi.avatar))
+                                is AvatarUi.Local -> {
+                                    // Local images are not uploaded yet, so we don't need to delete them
+                                }
+                            }
```

Comment:
This would be cleaned with the suggestion from the previous comment. We can pass a different object here, as we don't need this for `Local` avatars.